tea.go

```diff
@@ -116,7 +116,7 @@ type Program struct {
 	ignoreSignals      bool
 
 	// Stores the original reference to stdin for cases where input is not a
-	// TTY on windows and we've automatically opened CONIN$ to receive input.
+	// TTY. On Windows we've automatically opened CONIN$ to receive input.
```

Comment:
This change actually changes the meaning a bit. The original comment is accurate, this is for cases where the input is not a TTY on Windows, and we've automatically opened CONIN$ to receive input.

The comma was probably missing.